adam_core/observations/exposures.py

```diff
@@ -19,7 +18,7 @@ class Exposures(qv.Table):
     """
 
     id = qv.StringColumn()
-    start_time = Times.as_column()
+    start_time = Timestamp.as_column()
     duration = qv.Float64Column(validator=and_(ge(0), le(3600)))
```

Comment:
I wonder if switching this to an integer (nanoseconds?) would be worth the inconvenience?